using-vertx/src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
-quarkus.ssl.native=true
\ No newline at end of file
+quarkus.ssl.native=true
```

Comment:
any reason for SSL?

Reply:
I think it's due to Postgres connecting by default over SSL.

Reply:
I can't remember why exactly. I believe you added it when you participated in the "Using Vert.x" quickstart PR :smile_cat: